package.json

```diff
@@ -12,8 +12,8 @@
     "start": "NODE_ENV=production node build/server.js",
     "speculate": "speculate",
     "storybook": "start-storybook -p 9001 -c .storybook",
-    "test:e2e": "npx cypress run",
-    "test:e2e:interactive": "npx cypress open"
+    "test:e2e": "npm run dev & npx cypress run && pkill -f node",
```

Comment:
Could you confirm whether this would kill all ports running a node application? Is it possible for us to target an exact application EG: razzle?

Reply:
Yes it would kill all ports running a node application. I can update it to remove just the razzle processes
These are removed:
```
28408 ttys001    0:00.24 npm
28411 ttys001    0:00.11 node /dev/simorgh/node_modules/.bin/razzle start
28413 ttys001    0:06.26 node /dev/simorgh/node_modules/razzle/scripts/start.js
28423 ttys001    0:00.37 /.nvm/versions/node/v8.11.2/bin/node -r source-map-support/register
```